data/json/regional_map_settings.json

```diff
@@ -66,6 +66,7 @@
           "t_tree_birch": 16,
           "t_tree_elm": 16,
           "t_tree_cottonwood": 16,
+          "t_tree_basswood": 16,
```

Comment:
Would you mind adding these changes to data/mods/innawood/innawood_map_settings.json as well?
May or may not also want to add them to the following files. Not sure if it fit their respective mod's lore:
data/mods/rural_biome/rural_regional_map_settings.json
data/mods/desert_region/desert_regional_map_settings.json
data/mods/aftershock_exoplanet/region_settings.json
data/mods/classic_zombies/alberta_regional_map_settings.json
data/mods/TropiCataclysm/tropical_regional_map_settings.json
data/mods/No_Hope/regional_map_settings.json

Reply:
It seems fitting for No Hope, innawood, rural biome and classic zombies. I would not touch the other mods. I have thought about ignoring mods entirely for now, especially for actively developed mods like innawood because fitting it in there might be tricky in some cases. But I will think about it when I progress further.

Reply:
Also I thought about renaming the basswood leaves to more generic young leaves since apparently elm and other trees have edible young leaves, too.

Reply:
If this gets accepted to mainline, then, as the developer of innawood, I would much appreciate having the changes in my mod as well. Mirroring your changes from regional_map_settings.json to innawood_map_settings.json should make sure your new flora spawns in the mod too, and I don't think that would cause any issues.

Reply:
There are already a lot of problems with that. Since I know your mod has the lore of "humans never touched this land", why would there be invasive species from say asia?
When I consider this PR more or less done I will take a look at how much work integrating my content into the different mods will be.